src/constants/index.ts

```diff
@@ -7,6 +7,8 @@ type ChainTokenList = {
   readonly [chainId in ChainId]: Token[]
 }
 
+export const CAKE = new Token(ChainId.MAINNET, '0x0E09FaBB73Bd3Ade0a17ECC321fD13a19e81cE82', 18, 'CAKE', 'PancakeSwap Token')
```

Comment:
Export the constant to be able to use the `address` property for the API; instead of using the hardcoded `Cake` symbol